src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package duke.exception;
+
+public abstract class DukeException extends Exception {
+
+    public abstract String getResponseMessage();
```

Comment:
I feel like an exception class shouldn't be responsible for what is displayed to the user.